pom.xml

```diff
@@ -208,6 +209,24 @@
         <version>055ddfd5d6d9</version>
       </dependency>
 
+      <!-- tracing -->
+      <dependency>
```

Comment:
Is this dependency needed?

Reply:
Following the [instructions](https://cloud.google.com/trace/docs/setup/java)